frame/staking/src/migration.rs

```diff
@@ -111,6 +112,15 @@ pub mod inner {
 		}
 	}
 
+	// migrate storage from v2 to v3: Insert into CurrentEraStart the current time.
+	pub fn to_v3<T: Trait>(version: &mut VersionNumber) {
+		if *version != 2 { return }
+		*version += 1;
+
+		let now_as_millis_u64 = T::UnixTime::now().as_millis().saturated_into::<u64>();
+		<Module<T> as Store>::CurrentEraStart::put(now_as_millis_u64);
```

Comment:
We don't try to update from current era start because the moment might not be the same as UnixTime. (like it could be millisecond from the start of the chain)

But if we consider migration only for Kusama we could indeed do a translation.

Note: that this will decrease the duration of the current era, thus decreasing its reward